tests/unit_tests/parse/fortran/test_fortran_analyser.f90

```diff
@@ -17,6 +17,11 @@ SUBROUTINE internal_sub
         RETURN
     END SUBROUTINE internal_sub
 
+    SUBROUTINE openmp_sentinel
+!$ USE compute_chunk_size_mod, ONLY: compute_chunk_size  ! Note OpenMP sentinel
+!$ USE test that is not a sentinel with a use statement inside
+    END SUBROUTINE openmp_sentinel
```

Comment:
How about testing some other directives such as `!GCC$ unroll 6`, `!DIR$ assume (mod(p, 6) == 0)`, `!$omp do` and `!$acc parallel copyin (array, scalar)`.

Reply:
Done (though since these directives are ignored, there isn't much I can explicitly compare, except no new dependencies are added, so all that changed in the tests were the checksums)